qiita_ware/test/test_ebi.py

```diff
@@ -405,6 +410,72 @@ def test_generate_curl_command(self):
                          '%20ebi_access_key%3D"')
         self.assertEqual(obs, exp_with_cert)
 
+    def test_generate_demultiplexed_fastq(self):
+        # generating demux file for testing
+        if not isdir(self.temp_dir):
+            makedirs(self.temp_dir)
+
+        fna_fp = join(self.temp_dir, 'seqs.fna')
+        demux_fp = join(self.temp_dir, 'demux.seqs')
+        with open(fna_fp, 'w') as f:
+            f.write(FASTA_EXAMPLE)
+        with File(demux_fp, "w") as f:
+            to_hdf5(fna_fp, f)
```

Comment:
I just realized that these files go to the folder that you're also marking to remove.